docs/calendar-review.md

```diff
@@ -47,9 +47,29 @@ Examples of calendar-sensitive operations include:
 In other words, units of time greater than a solar day are inherently dependent on the calendar system.
 The only universal solution to date arithmetic would be to omit the concept of months and years, but a date API without months and years would be insufficient for many use cases.
 
+### Why are Calendars a Higher Priority for ECMAScript vs. Other Platforms?
+
+Problems integrating calendar systems into Temporal are obvious, especially the "[Unexpected Calendar Problem](#the-unexpected-calendar-problem)" where code breaks when presented with a calendar it wasn’t designed to handle.
+What's less clear are the benefits of integrating calendars in Temporal that can outweigh those problems.
+One way to understand these benefits is to highlight differences between the needs of the ECMAScript ecosystem compared to other platforms like Java and .NET which have taken a different approach to handling calendars.
+Some of these differences include:
+
+- The ECMAScript standard library is 1-2 orders of magnitude smaller than Java and .NET, which often leads to ECMAScript apps using 100+ OSS libraries (including transitive dependencies).
+  Because i18n in ECMAScript requires both platform _and_ library support, ECMAScript API patterns often nudge library developers to write globalized code, even if it requires somewhat more work.
+- As is well described in the [notes of Java's design of its date/time API](https://github.com/ThreeTen/threeten/wiki/Multi-calendar-system), the overwhelming majority of code in business apps (in Java and also ECMAScript too!) uses the ISO calendar.
+  But relative to Java/.NET, the ECMAScript ecosystem's center of gravity includes relatively more consumer and in-browser use cases.
+  Those use cases are where non-ISO calendar usage is concentrated.
+- Usage of non-ISO calendars is heaviest in the developing world which has historically had less software development targeted at local end-users.
+  Given these countries' continuing growth in smartphone adoption, middle-class incomes, and population, we expect increased demand for locally-specific tech in these markets, which in turn will drive more usage of non-ISO calendars.
+
+Choosing the current design over over the [alternatives considered](#alternatives-considered) was not an easy decision.
+As described below, this decision will cause more bugs in ISO-only software than other possible solutions.
+But, given global trends in economic growth, population growth, and tech adoption, the consensus of this proposal's champions is that the current design is a reasonable tradeoff for long-term success in an increasingly global ECMAScript developer community.
```

Comment:
```suggestion
### Why are Calendars a Higher Priority for ECMAScript vs. Other Platforms?

Problems integrating calendar systems into Temporal are obvious, especially the "Unexpected Calendar Problem" ([see below](#the-unexpected-calendar-problem)), where code breaks when presented with a calendar it wasn’t designed to handle.
What's less clear are the benefits of integrating calendars in Temporal that can outweigh those problems.
One way to understand these benefits is to highlight differences between the needs of the ECMAScript ecosystem compared to other platforms like Java and .NET which have taken a different approach to handling calendars.
Some of these differences include:

- As is well described in the [notes of Java's design of its date/time API](https://github.com/ThreeTen/threeten/wiki/Multi-calendar-system), the most code in enterprise apps uses the ISO calendar.
  But relative to Java/.NET, the ECMAScript ecosystem's center of gravity includes relatively more consumer and in-browser use cases.
  Those use cases are where non-ISO calendar usage is concentrated.
- Usage of non-ISO calendars is heaviest in emerging markets.  Given the rapid growth of smartphone adoption among the Next Billion Users, we expect increased demand for locally-specific tech in these markets, which in turn will drive more usage of non-ISO calendars.
- The ECMAScript standard library is 1-2 orders of magnitude smaller than Java and .NET, which often leads to ECMAScript apps using 100+ OSS libraries (including transitive dependencies).
  Because i18n in ECMAScript requires both platform _and_ library support, ECMAScript API patterns often nudge library developers to write globalized code, even if it requires somewhat more work.

Choosing the current design was not easy, because it will cause more bugs in ISO-only software than other possible solutions.
But, given global trends in economic growth, population growth, and tech adoption, the consensus of this proposal's champions is that the current design is a reasonable tradeoff for long-term success in an increasingly global ECMAScript developer community.
```